lib/provisioner/worker/plugins/automators/chef_solo_automator/resources/cookbooks/coopr_docker/metadata.rb

```diff
@@ -0,0 +1,11 @@
+name             'coopr_docker'
+maintainer       'Cask Data, Inc.'
+maintainer_email 'ops@cask.co'
+license          'Apache 2.0'
+description      'Simple Docker installer'
+long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
```

Comment:
`README.md` doesn't exist...

Reply:
Ha! Forgot to add it. I'll get that done.